server/src/main/java/org/elasticsearch/Version.java

```diff
@@ -167,7 +167,7 @@
     public static final Version V_6_3_0 = new Version(V_6_3_0_ID, org.apache.lucene.util.Version.LUCENE_7_3_0);
     public static final int V_7_0_0_alpha1_ID = 7000001;
     public static final Version V_7_0_0_alpha1 =
-        new Version(V_7_0_0_alpha1_ID, org.apache.lucene.util.Version.LUCENE_7_3_0);
```

Comment:
can you also replace the 7_2_1 version constant above to replace it with the one that comes from Lucene?

Reply:
Done - I pushed https://github.com/elastic/elasticsearch/pull/29398/commits/81a33ec3dfd8adbe6a4b54a8f37944d7a7645f52